UI/Selenium/Pages/WaitingRoomPage.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
```

Comment:
This does not look right!

Did you add this or did you find it to be something missing?

Reply:
may be this was added as a result of merge. now I have removed unused usings from this file